### PR TITLE
Gem foundation

### PR DESCRIPTION
**Pull request by julian-huff**
Basic support for both `/` and `/lists` to get the ball rolling.

### CODE REVIEW

**Comment by seashellski** [comment]
Squash the commits.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+###### GET `/lists`
+```ruby
+account.with_mailchimp_session { MailchimpAPI::List.all }
+```
+
+## Development
+
+After checking out the repository, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```
Comment (seashellski) [inline README.md:49]: `rails test` in Rails 5 land, no?

File: mailchimp_api.gemspec [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  spec.summary       = "Consume Mailchimp's API using ActiveResource"
+  spec.description   = "Consume Mailchimp's API using ActiveResource"
+  spec.homepage      = 'https://github.com/rewindio/mailchimp_api.git'
+  spec.license       = 'MIT'
+
+  spec.require_paths = %w[lib]
+
+  spec.files = `git ls-files -z lib`.split("\x0") + %w[CHANGELOG.md LICENSE README.md]
+
+  spec.add_runtime_dependency 'activeresource', '~> 5.1.0'
```
Comment (seashellski) [inline mailchimp_api.gemspec:23]: image

**Review by seashellski — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (julian-huff) [reply README.md:49]: Not a rails app bro

**Comment by seashellski** [comment]
General comment for open source, should we have a style guide? Or do you plan on rejecting PRs for bracket and tab violations?

**Comment by julian-huff** [comment]
I think Rubocop should be good enough

**Review by smandegar — APPROVED** [review]

**Pull request merged**